OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -434,6 +435,40 @@ internal sealed class DreamInterfaceManager : IDreamInterfaceManager {
         });
     }
 
+    public void RunCommand(string command){
+        switch (command) {
+                case string x when x.StartsWith(".quit"):
+                    IoCManager.Resolve<IClientNetManager>().ClientDisconnect(".quit used");
+                    break;
+
+                case string x when x.StartsWith(".screenshot"):
+                    string[] split = command.Split(" ");
+                    SaveScreenshot(split.Length == 1 || split[1] != "auto");
+                    break;
+
+                case string x when x.StartsWith(".configure"):
+                    _sawmill.Warning(".configure command is not implemented");
+                    break;
+
+                case string x when x.StartsWith(".winset"):
+                    // Everything after .winset, excluding the space and quotes
+                    string winsetParams = command.Substring(7); //clip .winset
+                    winsetParams = winsetParams.Trim(); //clip space
+                    winsetParams = winsetParams.Trim('\"'); //clip quotes
+
+                    WinSet(null, winsetParams);
+                    break;
+
+                default: {
+                    // Send the entire command to the server.
+                    // It has more info about argument types so it can parse it better than we can.
+                    //RaiseNetworkEvent(new CommandEvent(command));
+                    EntitySystem.Get<DreamCommandSystem>().SendServerCommand(command);
```

Comment:
This kinda breaks the whole point of moving this to `DreamInterfaceManager`. Moving it here so you can use it before entity systems load won't work if you still attempt to get the entity system anyways.

You'll have to replace the network event with a custom `NetMessage`.